ext/pdo_pgsql/php_pdo_pgsql_int.h

```diff
@@ -86,6 +86,7 @@ extern const struct pdo_stmt_methods pgsql_stmt_methods;
 
 enum {
 	PDO_PGSQL_ATTR_DISABLE_PREPARES = PDO_ATTR_DRIVER_SPECIFIC,
+	PRO_PGSQL_ATTR_RESULT_MEMORY_SIZE,
```

Comment:
ditto

```suggestion
	PDO_PGSQL_ATTR_RESULT_MEMORY_SIZE,
```

Reply:
Fixed in https://github.com/php/php-src/pull/14260/commits/a420d3afda36fc0789dfd7e56deb72d37ae3854e